File: Erik_Vincent/Assignment1/Assignment1/Model/QLS/AST/Style/Widget/Slider.cs
using Assignment1.Model.QL.AST;

namespace Assignment1.Model.QLS.AST.Style.Widget
{
    public class Slider : ASTNode, IWidget
    {
        public Slider(int lineNumber)
        {
            _lineNumber = lineNumber;
        }

        public void Accept(IStyleVisitor visitor) => visitor.Visit(this);
    }
}

File: GunOli/src/main/java/QLS/AST/Widgets/Slider.java
package QLS.AST.Widgets;

import QLS.Analysis.WidgetVisitorInterface;

public class Slider extends Widget {

    public Slider(int line) {
        super(WidgetType.Slider, line);
    }

    @Override
    public <T> T accept(WidgetVisitorInterface<T> visitor) {
        return visitor.visit(this);
    }
}
